src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -6,6 +6,9 @@
 
 #include <absl/container/flat_hash_set.h>
 #include <absl/flags/declare.h>
+#include <absl/flags/internal/flag.h>
+#include <qgridlayout.h>
+#include <qnamespace.h>
```

Comment:
Do you need these here? I think we don't usually include `<q... .h>` headers, or do we?

Reply:
Thanks for flagging, those were leftover from some experimentation... Removed.